howdoi/howdoi.py

```diff
@@ -512,7 +526,7 @@ def print_stash(stash_list=None):
         stash_list = ['\nSTASH LIST:']
         commands = keep_utils.read_commands()
         if commands is None or len(commands.items()) == 0:
-            print(f'No commands found in stash. Add a command with "howdoi --{STASH_SAVE} <query>".')
+            debug_logging(f'No commands found in stash. Add a command with "howdoi --{STASH_SAVE} <query>".')
```

Comment:
I believe we'll want to use a different log level here. Previously, the print statement would always be shown when trying to view and empty stash. With this PR it isn't shown anymore when you try:

```
python -m howdoi.howdoi --empty
python -m howdoi.howdoi --view
```

It could be an error log?